src/nvim/mouse.c

```diff
@@ -189,58 +191,70 @@ int jump_to_mouse(int flags, bool *inclusive, int which_button)
   old_curwin = curwin;
   old_cursor = curwin->w_cursor;
 
-  if (!(flags & MOUSE_FOCUS)) {
-    if (row < 0 || col < 0) {                   // check if it makes sense
-      return IN_UNKNOWN;
-    }
+  if (row < 0 || col < 0) {                   // check if it makes sense
+    return IN_UNKNOWN;
+  }
```

Comment:
the docstring explains the `if (!(flags & MOUSE_FOCUS))` guard as follows, is this still the behavior?

```
/// If flags has MOUSE_FOCUS, then the current window will not be changed, and
/// if the mouse is outside the window then the text will scroll, or if the
/// mouse was previously on a status line, then the status line may be dragged.
```

Reply:
yes, unless the statusline or winbar was clicked, then there will be no scroll